templates/patterns/featured_media/featured_media.ui_patterns.yml

```diff
@@ -35,6 +35,14 @@ featured_media:
       label: "Heading"
       description: "The heading, displayed on the top."
       preview: "Lorem ipsum"
+    title_tag:
+      type: text
+      label: 'Title tag'
+      description: 'The tag to use for the title. Defaults to h2.'
+    title_link:
+      type: LinkObject
```

Comment:
The type must be a php type, like string, array, or an object of a certain class name, this is neither. Let's use string and set whatever format the heading in the template. Then also name it correspondingly, perhaps title_url ?

Reply:
Indeed. I also added tests and changed the pattern of gallery. Please have a look.  I guess we either change the ticket and title of this PR or we make the gallery changes in another ticket.